suseconnect-ng.spec

```diff
@@ -108,6 +108,15 @@ find %_builddir/..
 echo %{version} > internal/connect/version.txt
 %goprep %{import_path}
 find %_builddir/..
+# go1.18+ default behavior has runtime soft dependency on git, hg, etc.
+# When VCS e.g. .git/ detected in sources 'go build', 'go list'
+# and related subcommands attempt to stamp/read vcs info.
+# As the VCS info is potentially useful for software supply chain,
+# prefer not to override defaults globally via golang-packaging at this time.
+# For now, suppress in each build section before go tool use,
+# or alternatively add BuildRequires dependency on git.
+# Also see 'go version -m ./binaryname' as alternative build info.
+export GOFLAGS="-buildvcs=false"
```

Comment:
This fails on go 1.17 with `go: parsing $GOFLAGS: unknown flag -buildvcs`

Reply:
A rebased commit has been pushed with conditional checks using `rpm_vercmp` and `go env GOVERSION` as the argument. This works for both older and current versions of go.